### PR TITLE
Fix/broken menu auctions

### PR DESCRIPTION
**Pull request by W3stside**
# Attempt 1 at MenuAuctions bug fix

1. made testing prod vs dev easier in local
2. SCSS/code clean
3. Fixed MenuAuctions grabbing prop from null Auction object
  a. Created `bothTheoClosed` var to represent edge case state

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

File: src/api/index.ts [hunk 1]
```diff
@@ -473,7 +473,7 @@ export const getOutstandingVolume = async (
     price || DutchX.getPrice(pair, auctionIndex),
   ])
 
-  const outstandingVolume = sellVolume.mul(price[0]).div(price[1]).sub(buyVolume)
+  const outstandingVolume = price[1].gt(0) ? sellVolume.mul(price[0]).div(price[1]).sub(buyVolume) : toBigNumber(0)
```
Comment (Velenir) [inline src/api/index.ts]: Do a shortcut:
```
if (price[1].eq(0)) return toBigNumber(0)

const outstandingVolume = sellVolume.mul(price[0]).div(price[1]).sub(buyVolume)
```

File: src/api/index.ts [hunk 2]
```diff
@@ -921,8 +923,9 @@ export const getSellerOngoingAuctions = async (
         auctionStart,
       } = lastAuctionsData[index]
 
-      const currAuctionNeverRanDir = balanceNormal.eq(0) && closingPriceDir[1].eq(0)
-      const currAuctionNeverRanOpp = balanceInverse.eq(0) && closingPriceOpp[1].eq(0)
+      const bothAuctionTheoClosed = statusDir.theoreticallyClosed && statusOpp.theoreticallyClosed
+      const currAuctionNeverRanDir = bothAuctionTheoClosed || (balanceNormal.eq(0) && closingPriceDir[1].eq(0))
```
Comment (Velenir) [inline src/api/index.ts]: Direct auction had balance, which was breaking, or was it the opposite?

File: src/api/contracts.ts [hunk 3]
```diff
@@ -49,8 +49,11 @@ interface ContractsMapWProxy extends ContractsMap {
   TokenOWLProxy: DeployedContract,
 }
 
+// TODO: keep as 'development'
+const FE_CONDITIONAL_ENV = 'development'
```
Comment (Velenir) [inline src/api/contracts.ts]: what for?

File: src/api/index.ts [hunk 4]
```diff
@@ -754,6 +754,8 @@ const getLastAuctionStats = async (DutchX: DutchExchange, pair: TokenPair, accou
 
   if (auctionStart.eq(1)) statusDir.status = statusOpp.status = AuctionStatus.INIT
   else {
+    // outstandingVolume <= 0 === THEORETICALLY CLOSED
+    // TODO: ask why num here and not den
     if (closingPriceDir[0].equals(0) && outstandingVolumeNormal.eq(0)) {
```
Comment (Velenir) [inline src/api/index.ts:760]: doesn't matter, can be den

**Review by Velenir — CHANGES_REQUESTED** [review]

File: src/api/contracts.ts (hunk 3 above)
Comment (W3stside) [reply src/api/contracts.ts]: testing without `npm run start:prod` and keeping logging

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

**Review by Velenir — APPROVED** [review]

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr390--dxreact.review.gnosisdev.com

**Pull request merged**